addons/web/static/src/webclient/actions/reports/report.scss

```diff
@@ -90,6 +90,18 @@ span.o_force_ltr {
     border-top: 1px solid black !important;
 }
 
+.border {
+    border: 1px solid #dee2e6;
+}
+
+.border-black {
+    border-color: #000000;
+}
+
+.border-danger {
```

Comment:
isn't it "border-dark" for the border around RUT number?

> ![2024-10-07-131937_953x645_scrot](https://github.com/user-attachments/assets/91a1e62b-06c0-4396-8d5b-af954e15a0b5)

on the runbot I see it in light-grey instead of black

also .border-danger doesn't seem used but maybe it's for another thing

Reply:
> isn't it "border-dark" for the border around RUT number?

It is... border-black doesn't exist in bootstrap https://getbootstrap.com/docs/4.0/utilities/borders/

Reply:
> isn't it "border-dark" for the border around RUT number?

Ah thanks, I forgot the one that I was trying to fix :sweat_smile: 

>also .border-danger doesn't seem used in that part but maybe it's for another thing

I added the most common border.* definitions to prevent further tickets on this
I tried to directly backport https://github.com/odoo/odoo/blob/18.0/addons/web/static/src/webclient/actions/reports/utilities_custom_report.scss but it's not working as expected :thinking: